### PR TITLE
Provide scopes for custom node operations

### PR DESCRIPTION
**Pull request by ivov**
```
  OAuth2 scopes
    ✓ GET /oauth2-credential/scopes should return scopes - comma-delimited (56 ms)
    ✓ GET /oauth2-credential/scopes should return scopes - whitespace-delimited (29 ms)
    ✓ GET /oauth2-credential/scopes should return scope - non-delimited (31 ms)
    ✓ GET /oauth2-credential/scopes should fail with missing credential type (25 ms)
    ✓ GET /oauth2-credential/scopes should fail with non-OAuth2 credential type (11 ms)
    ✓ GET /oauth2-credential/scopes should fail with wrong credential type (24 ms)
```

### CODE REVIEW

File: packages/cli/src/Server.ts [hunk 1]
```diff
@@ -1932,6 +1933,8 @@ class App {
 		// OAuth2-Credential/Auth
 		// ----------------------------------------
 
+		this.app.use(`/${this.restEndpoint}/oauth2-credential`, oauth2CredentialController);
```
Comment (ivov) [inline packages/cli/src/Server.ts:1936]: In future we can move `/oauth2-credential/auth` and `/oauth2-credential/callback` to the new controller.

File: packages/cli/src/LoadNodesAndCredentials.ts [hunk 2]
```diff
@@ -58,6 +58,7 @@ class LoadNodesAndCredentialsClass {
 			// In case "n8n" package is the root and the packages are
 			// in the "node_modules" folder underneath it.
 			path.join(__dirname, '..', '..', 'node_modules', 'n8n-workflow'),
+			path.join(__dirname, '..', 'node_modules', 'n8n-workflow'), // for test run
```
Comment (ivov) [inline packages/cli/src/LoadNodesAndCredentials.ts:61]: Not ideal, but larger refactoring would seem out of scope.

File: packages/cli/src/Server.ts (hunk 1 above)
Comment (BHesseldieck) [reply packages/cli/src/Server.ts:1937]: we should 😊 can you create a Tech Debt ticket for it?

**Review by BHesseldieck — APPROVED** [review]
